src/supervisor2.erl

```diff
@@ -1438,7 +1438,7 @@ validMods(Mods) -> throw({invalid_modules, Mods}).
 %%% Add a new restart and calculate if the max restart
 %%% intensity has been reached (in that case the supervisor
 %%% shall terminate).
-%%% All restarts accured inside the period amount of seconds
+%%% All restarts accrued inside the period amount of seconds
```

Comment:
I'm not sure about this

Reply:
That sentence is broken in more than one way. I think it should say "all restarts occur".

Reply:
I've changed it to occurring in 2ff7143